File: 프로그래머스/0/181915. 글자 이어 붙여 문자열 만들기/글자 이어 붙여 문자열 만들기.java
class Solution {
    public String solution(String my_string, int[] index_list) {
        char[] ch = new char[index_list.length]; 
        
        for (int i = 0; i < index_list.length; i++) {
            ch[i] = my_string.charAt(index_list[i]);
        }
        
        String answer = new String(ch);
        return answer;
    }
}

File: 프로그래머스/0/181915. 글자 이어 붙여 문자열 만들기/글자 이어 붙여 문자열 만들기.cs
using System;

public class Solution {
    public string solution(string my_string, int[] index_list) {
        char[] ch = new char[index_list.Length]; 
        
        for (int i = 0; i < index_list.Length; i++) {
            ch[i] = my_string[index_list[i]];
        }
        
        String answer = new String(ch);
        return answer;
    }
}